shell/platform/android/image_external_texture_vk.cc

```diff
@@ -57,8 +59,33 @@ void ImageExternalTextureVK::ProcessFrame(PaintContext& context,
           desc, impeller_context_->GetDevice(), latest_hardware_buffer,
           hb_desc);
 
-  auto texture =
-      std::make_shared<impeller::TextureVK>(impeller_context_, texture_source);
+  auto texture = std::make_shared<impeller::TextureVK>(impeller_context_,
```

Comment:
This is new. When consuming the external texture we transition it eagerly instead of in the render pass.

Reply:
I talked to @johnmccutchan and he said that we only use these textures as sampled images, so this should be safe.